src/lib/components/NeovimConfigInstall.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+	import { faCopy } from '@fortawesome/free-solid-svg-icons';
+	import Fa from 'svelte-fa';
+
+  export let owner;
+  export let repo;
+
+  const code = `
+git clone https://github.com/${owner}/${repo}.git ~/.config/${owner}/${repo}
+NVIM_APPNAME=~/.config/${owner}/${repo} nvim`;
```

Comment:
We need to use `NVIM_APPNAME=~/.config/${owner}/${repo}/${root} nvim` I believe in order to get to the nvim directory if they use a mono repo